Add client depndency injection missing (#15664)
demisto/content: Packs/Anomali_ThreatStream/ReleaseNotes/2_0_1.md

#### Integrations
##### Anomali ThreatStream v2

> - Fixed an issue where the *!threatstream-submit-to-sandbox* command would show an error even when it was successful.

Reviewer: ```suggestion
- Fixed an issue where the ***threatstream-submit-to-sandbox*** command returned an error even when it ran successfully.
```
